### PR TITLE
changing default location of api routes

### PR DESCRIPTION
**Pull request by mireille-raad**

### CODE REVIEW

File: routes/api.php [hunk 1]
```diff
@@ -16,3 +16,6 @@
 Route::middleware('auth:api')->get('/user', function (Request $request) {
     return $request->user();
 });
+
+
+Route::get('sirup/{year}', 'ApiBIRMS@sirupAll');
```
Comment (bdgcoder) [inline routes/api.php:21]: Whats this

**Review by bdgcoder — COMMENTED** [review]
Just ask

File: routes/api.php (hunk 1 above)
Comment (mireille-raad) [reply routes/api.php:21]: testing routes

Comment (bdgcoder) [reply routes/api.php:21]: ok

**Pull request merged**